django/db/models/fields/files.py

```diff
@@ -75,6 +75,7 @@ def open(self, mode='rb'):
             self.file.open(mode)
         else:
             self.file = self.storage.open(self.name, mode)
+        return self
```

Comment:
Okay. I've added a test at https://github.com/ingokloecker/django/commits/ticket_27777. How does it get merged? Do I create a new pull request?